### ISSUE TITLE
[Bug] training with data augumentation  is very slow

### ISSUE THREAD

**Issue by HaoLiuHust** [open-mmlab/mmpretrain#1824]
### Branch

main branch (mmpretrain version)

### Describe the bug

I am using mmpretrain to train a mutli label model, I am using 16 core cpu and 2 3090 gpus, but the training speed is too slow, it cost almost 2 days compared my orignal code which runned on 2 1080Ti cost only 6 hours. The data loader stuck the speed, I have used 8 workers per gpu, but I found the cpu usage is very slow, what can I do to speed up it?

### Environment

'CUDA_HOME': '/usr/local/cuda',
 'NVCC': 'Cuda compilation tools, release 11.7, V11.7.99',
 'GCC': 'gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0',
 'PyTorch': '2.0.1',
 'TorchVision': '0.15.2',
 'OpenCV': '4.8.0',
 'MMEngine': '0.8.4',
 'MMCV': '2.0.1',
 'MMPreTrain': '1.1.0+'}

**Comment by HaoLiuHust** [comment]
set opencv_num_threads to 2 solved the problem
